frontend/src/pages/blog/CreateArticle.js

```diff
@@ -51,15 +52,19 @@ const CreateArticle = () => {
 
   const handleFileChange = (event) => {
     const file = event.target.files[0];
-    console.log(file);
-    console.log(file.text());
-    file.text().then((t) => {
-      const lines = t.split("\n");
-      console.log(lines);
-      setTitle(lines[0]);
-      setSummary(lines[1]);
-      setBody(lines.slice(3).join("\n"));
-    });
+    console.log(file.size)
```

Comment:
Is using console.log a good habit by the way ?
You could be a little bit explicit or do you prefer waiting for proper logging ?

Reply:
The `console.log` is just logging for me when coding.
And yes the real logs will come with #10